.github/mergify.yml

```diff
@@ -0,0 +1,10 @@
+pull_request_rules:
+  - name: Automatic merge on approval
+    conditions:
+      - and:
+        - "-draft"
+        - "#check-failure=0"
+        - "#approved-reviews-by>=1"
```

Comment:
I don't know whether we want to auto-merge on the first approval. At the moment I merge trivial things right away will I leave other things I approve open for a while to give other the chance to give a look.

Reply:
This did indeed cause #1477 to be merged by accident